src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -93,6 +93,8 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
     static _setTextProp (el, propName, text) {
         const processedText = text !== null && text !== void 0 ? String(text) : text;
 
+        DOMMutationTracker.onChildrenAddedOrRemoved(el);
```

Comment:
Why do we call this before element's children is changed?
Where and below in code.